INFWS-182 Draft GUI App usage notes (#2954)
chef/chef-workstation: docs-chef-io/content/workstation/chef-workstation-app.md

Start `Chef Workstation App` from your Applications folder.

> ## Managing cookbooks

Reviewer: What does this UI let you do and what will it do? We should probably start there. Also, is there a name for this window? It looks like we have two things, the menu in the taskbar and the window UI. The menu in the taskbar says "Manage Cookbooks", but the UI window title is Chef Workstation App, which might lead to confusion if this is only to handle Cookbooks.

It might make sense to start with a heading for the window UI. 

So something like:

> ## Chef Workstation App Window

> The Chef Workstation App window lets you manage Cookbooks and Chef repositories. Open the Chef Workstation App window by selecting **Manage Cookbooks** from the Chef Workstation App menu in the menu bar (macOS) or taskbar (Windows).

Author: Updated as suggested. Thank you!

> ## Chef Workstation App Window
> 
> The Chef Workstation App window lets you manage Cookbooks and Chef repositories. Open the Chef Workstation App window by selecting **Manage Cookbooks** from the Chef Workstation App menu in the menu bar (macOS) or taskbar (Windows).

Reviewer: I notice you're using the term "dashboard" below. Whether it's window or dashboard, we should just be consistent.

```suggestion
## Chef Workstation App Dashboard

The Chef Workstation App dashboard lets you manage Cookbooks and Chef repositories. Open the Chef Workstation App dashboard by selecting **Manage Cookbooks** from the Chef Workstation App menu in the menu bar (macOS) or taskbar (Windows).
```

> ### Linking repositories
> 
> The 'Chef Repositories' view lets you to choose folders in your computer that represent the chef-repo's you need to work on. Click 'Add New' to select a chef-repo folder. The cookbooks present in each of these repositories will be listed in the dashboard's 'Cookbooks' view. The linked chef-repo's are persisted so you only need to add them once in the cookbook management dashboard. The repository name displayed in the dashboard is the same as the folder name.

Reviewer: ```suggestion
### Chef Repositories

Select **Chef Repositories** to view Chef repositories and cookbooks on your local workstation.

To add a new Chef repository and its cookbooks, select **Add New** and then navigate to a Chef repository in your workstation's file system.

The linked Chef repositories are persistent, so you only need to add them once in the dashboard. The repository name displayed in the dashboard is the same as the Chef repository folder name.
```

> ### Accessing cookbook actions
> 
> At present, the 'Cookbooks' view lets you upload the selected cookbooks to the configured Chef Infra Server. To upload a cookbook, click the 'Upload' button corresponding to the cookbook.

Reviewer: ```suggestion
### Cookbooks

Use the **Cookbooks** panel to upload cookbooks to a configured Chef Infra Server.

To upload a cookbook, select **Upload** next to the corresponding cookbook.
```

{{< note >}}

> There is a known limitation where the profile settings in `~/.chef/credentials` are not parsed correctly, causing a request signing error to show up when you upload a cookbook. To overcome this issue, please copy your knife `config.rb` to `~/.chef`. This is a temporary workaround and a proper fix will be introduced soon.

Reviewer: ```suggestion
There is a known limitation where the profile settings in `~/.chef/credentials` are not parsed correctly, causing a request signing error to show up when you upload a cookbook. To overcome this issue, copy your knife `config.rb` to `~/.chef`.
```
